ftag/wps/working_points.py

```diff
@@ -60,6 +60,13 @@ def parse_args(args):
         type=str,
         help='signal flavour ("bjets" or "cjets")',
     )
+    parser.add_argument(
+        "-r",
+        "--rejection",
+        default=None,
+        choices=["ujets", "cjets", "bjets"],
+        help="use rejection instead of efficiency",
+    )
```

Comment:
I think it would be better to have the `--rejection` argument specify the rejections directly, and make it exclusive with `-e `. Then we can just overload `--signal` (perhaps instead renaming to `--flavour` or `--target` to either be the signal class (in the case where `-e` is specified) or the background class (if `-r` is specified)

Reply:
Hmmm, but given that we in general have multiple background classes don't we still need to have two flags. One to set which class to calculate the discriminant with and one to set which we calculate the WP with?

Reply:
oops sorry yeah I realised this and posted by accident. What you have already is fine with the below changes.